extensions/vscode/webviews/homeView/src/axios.d.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright (C) 2023 by Posit Software, PBC.
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { AxiosRequestConfig } from "axios";
+
+declare module "axios" {
```

Comment:
I needed the second file so the homeView TypeScript compilation could grab the `.d.ts` file.